### PR TITLE
Fix Pylint step

### LINKED ISSUE #18: Fix workflow action error in Pylint step
## DESCRIPTION
There's an error with the workflow actions where the pylint is checking for a specific file called *.py instead of recognising as a wildcard character to search for all python file.
##USER STORY
As a data engineer, I need the workflow action to run the steps properly so that other engineers can properly push their code and have automated tests work properly for their codes.

### PR DESCRIPTION
**Pull request by punima-s**
Close #18

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ jobs:
       - name: "Run Pytest"
         run: pytest
       - name: "Run PyLint"
-        run: pylint --fail-under=1 *.py
+        run: pylint --fail-under=1 $(find . -name "*.py")
```
Comment (fm1psy) [inline .github/workflows/main.yaml:27]: I would like some clarification on what this change will do to the job.

**Review by fm1psy — COMMENTED** [review]
Hello, I left you a question as I just wanted to double check how the change you made is supposed to fix the pylint.

File: .github/workflows/main.yaml (hunk 1 above)
Comment (punima-s) [reply .github/workflows/main.yaml:27]: This finds python files instead of *.py file

**Review by fm1psy — APPROVED** [review]
Great, good to go.

**Pull request merged**